packages/big-design/src/components/FileUploader/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface Localization {
```

Comment:
All the `FileUploader` changes where just shifting things around, simplifying types (one `Localization` type), and removing type exports that weren't needed. Confirmed with @bc-apostoliuk which types were needed via export.